resources/views/recipes/single-recipe.blade.php

```diff
@@ -1,19 +1,42 @@
 @extends('layouts.app', ['vue' => 'single-recipe'])
 
 @section('content')
-    <h1>{{$recipe->title}}</h1>
-    <ul>
+
+<div class="recipe-wrapper">
+    <h2 class="page-title">{{$recipe->title}}</h2>
+    <h6 class="recipe-cat">Categories: <a>Lunch</a>, <a>Dinner</a></h6>
+    <nav class="mini-nav">
+        <ul>
+            <li><a><i class="fa fa-cart-plus"></i></a></li>
+            <li><a><i class="fa fa-pencil"></i></a></li>
+            <li><a><i class="fa fa-users"></i></a></li>
+            <li><a><i class="fa fa-trash"></i></a></li>
+        </ul>
+    </nav>
+
+    <h3 class="form-heading">Ingredients</h3>
+    <ul class="ingredients">
         @foreach($recipe->items as $item)
             <li>{{$item->quantity}} {{$item->name}}</li>
         @endforeach
     </ul>
-    <h1>Add to Grocery List</h1>
-    <ul>
-    @foreach($listsWithoutRecipe as $grocerylist)
-        {{--<li @click="addToGroceryList({{$grocerylist->getKey()}})">{{$grocerylist->title}}</li>--}}
-        <li><a href="/grocerylist/{{$grocerylist->getKey()}}/add/{{$recipe->getKey()}}">{{$grocerylist->title}}</a></li>
-    @endforeach
+
+    <h3 class="form-heading">Directions</h3>
+
+
+
+    <h5 class="add-list-title page-title">Add to Grocery List</h5>
+    <ul class="lists">
+        @foreach($listsWithoutRecipe as $grocerylist)
+            {{--<li @click="addToGroceryList({{$grocerylist->getKey()}})">{{$grocerylist->title}}</li>--}}
```

Comment:
That's your commented code (or at least it was commented when I started working on this page), not mine so I just left it there.